brainstormforce/astra
# AST-2793 - fix: code block margin bottom added to match frontend spacing.

### Description

- Margin below code-block is missing in editor.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2793
### Screenshots

https://d.pr/B6Gn7y
### Types of changes


bugfix: Margin below code-block is missing in editor.



### How has this been tested?

- Create a new post/page in editor.
- Create a code block.
- Code block should have margin, same as frontend. Earlier there was no margin.

### Checklist:
